packages/aws-rfdk/lib/deadline/lib/version.ts

```diff
@@ -22,6 +22,11 @@ export class Version implements IPatchVersion {
    */
   public static readonly MINIMUM_SECRETS_MANAGEMENT_VERSION = new Version([10, 1, 19, 0]);
 
+  /**
+   * The minimum Deadline version which uses Install Builder instead of Bitrock Installer.
+   */
+  public static readonly MINIMUM_VERSION_USING_INSTALLBUILDER = new Version([10, 2, 0, 0]);
```

Comment:
nit: Bitrock is the company that makes InstallBuilder. Both versions of Deadline use InstallBuilder to make installers, but I suspect that the version of InstallBuilder was changed between Deadline releases and that InstallBuilder changed its (default?) log file path.

Can we instead rename to `MINIMUM_VERSION_USING_NEW_INSTALLBUILDER_LOG` and adjust the code comment accordingly?

Reply:
Implemented in latest commit